src/main/java/core/basesyntax/store/StorageService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.store;
+
+public interface StorageService {
+    void makeOperationDependsOnType(String[] line, int lineNumber);
+}
```

Comment:
```suggestion
public interface StorageService {
    void saveData(List<FruitRecord> records);
    
    String getReport();
}
```

in the first method - you will get information about the operations with fruit and save changes to storage using the strategy pattern (handlers)

in the second method - you will get information about all fruit from storage and return the report as String (with header and information about all fruit)